pkgs/data/misc/shared-mime-info/default.nix

```diff
@@ -1,13 +1,13 @@
 {stdenv, fetchurl, pkgconfig, gettext, perlPackages, intltool
 , libxml2, glib}:
 
-let version = "1.10"; in
+let version = "1.12"; in
 stdenv.mkDerivation rec {
   name = "shared-mime-info-${version}";
 
   src = fetchurl {
```

Comment:
Maybe we should switch to `fetchFromGitLab` to avoid the hash.

Reply:
Possibly, only difference in that archive that noticed (after glancing) was that it had tests.

Reply:
Nah, doesn't look like anyone else bothers

* https://src.fedoraproject.org/rpms/shared-mime-info/blob/master/f/shared-mime-info.spec
* https://git.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/shared-mime-info

Reply:
The benefit would be the ability to use `common-updater-scripts`.